recipes/org/apache/httpcomponents/_artifact/httpclient/scm.yaml

```diff
@@ -0,0 +1,6 @@
+---
+type: "git"
+uri: "https://github.com/apache/httpcomponents-client.git"
+tagMapping:
+  - pattern: 4.5.13
```

Comment:
This should be a regex, so it applies to all versions. I will merge and update it with an example.

Reply:
https://github.com/redhat-appstudio/jvm-build-data/commit/d1fc98017d71a63999b6a9938a10a36a24537647